contoso_manufacturing/developer/webapp-decode/app.py

```diff
@@ -4,47 +4,105 @@
 import json
 import numpy as np
 from yolov8 import YOLOv8OVMS
-
-
+from welding import WeldPorosity
+from pose_estimator import PoseEstimator
 
 app = Flask(__name__)
 
-OVMS_URL = "10.0.0.4:8080"
-MODEL_NAME = ''
-
 camera = None  # Initialized later based on the selected video
+latest_choice_detector = None # Global variable to keep track of the latest choice of the user
+ovms_url = "192.168.0.4:31640"
+frame_number = 0
+skip_mod = 2 # Define the modulus to skip frames
 
-# Global variable to keep track of the latest choice of the user
-latest_choice = None
+# Init the config.file.json
+with open('config_file.json') as config_file:
+    config = json.load(config_file)
```

Comment:
Change to get the config.file json from the Kubernetes configMap already being used for OVMS